### PR TITLE
add footer component.

### PR DESCRIPTION
**Pull request by Ali7040**
# Angular App version

- [ ] `angular-v17` - has no UI libraries
- [ ] `angular-v17-modMAT` - modules and Angular Material
- [x] `angular-v17-AnguMAT` - has Angular Material
- [ ] `angular-v17-PrimeNG` - has PrimeNG 
- [ ] `angular-hashnode`- presentation site

## What type of PR is this? (check all applicable)
- [x] 🅰️ Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings
[Attach screenshots or recordings if applicable]

## Added to documentation?
- [ ] 📜 README.md
- [x] 🙅 no documentation needed

## [Optional] Post-deployment tasks
[Specify any post-deployment tasks that need to be performed]

## [Optional] What gif best describes this PR or how it makes you feel? 
[Embed gif or describe the feeling in plain text]

### CODE REVIEW

File: angular-v17-AnguMAT/src/app/components/footer/footer.component.html [hunk 1]
```diff
@@ -1 +1,12 @@
-<p>footer works!</p>
+<footer class="footer">
```
Comment (monacodelisa) [inline angular-v17-AnguMAT/src/app/components/footer/footer.component.html:1]: @Ali7040 I will merge it but just a quick reminder - the whole idea of the Angular Material version is to use angular material components, here I will later on replace <footer> with <mat-toolbar>

Comment (Ali7040) [reply angular-v17-AnguMAT/src/app/components/footer/footer.component.html]: okay, I will change it

**Review by Ali7040 — COMMENTED** [review]
@monacodelisa I've updated the code. If there are any other issues, please let me know.

**Comment by monacodelisa** [comment]
@Ali7040 this has already been implemented

**Comment by monacodelisa** [comment]
@Ali7040 I am adding your code, what I did was the presentation site which has pretty much identical code,
I have also moved the presentation site to a different repo 

Thank you for the contributions 👍

**Pull request merged**